src/frontend/pages/App.css

```diff
@@ -1,11 +1,39 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@400;700&display=swap');
+
+:root {
+	--font-family: 'Nunito', sans-serif;
+	--font-normal: 400;
+	--font-bold: 700;
+	--font-primary-color: #212121;
+	--font-secondary-color: #fff;
+	--background-color: #dcdcdc;
+    --page-max-width: 1200px;
+}
+
 html {
-    font-family: -apple-system, "BlinkMacSystemFont", "Segoe UI", "Roboto", "Helvetica Neue", "Arial", "Noto Sans",
-        sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
-    font-size: 20px;
+	font-family: var(--font-family);
+	font-size: 16px;
 }
 
 body {
-    position: relative;
-    color: black;
-    font-weight: 200;
+    display: flex;
+    flex-direction: column;
+	position: relative;
+    min-height: 100vh;
+	color: var(--font-primary-color);
+	font-weight: var(--font-normal);
+	background-color: var(--background-color);
```

Comment:
Seems like there's something strange going on with tabs and spaces at the beginnings of lines. Can you double-check they all use spaces?